config/application.yaml

```diff
@@ -26,8 +26,8 @@ quarkus:
 # Test profile
```

Comment:
I don't quite understand the role of this file, are these defaults or something? Since this appears to be config that would now be taken from environment variables?

Reply:
There are some config items such as mappings between URIs and local filestores that don't need to be so dynamic and may be OK to leave in this config file. I started with this Quarkus config mechanism before I had an application wrapper and before I was running in docker. Once we get further with the design of where annotations and test cases are stored, this needs reviewing.